### PR TITLE
[Controller] exposing registered depositories counters

### PR DESCRIPTION
**Pull request by crypto-vincent**
Exposing the registered counts in the controller parser

Used by: https://github.com/UXDProtocol/uxd-backoffice/pull/44

### CODE REVIEW

File: src/interfaces.ts [hunk 1]
```diff
@@ -8,10 +8,12 @@ export class ControllerAccount {
   authority!: PublicKey;
   redeemableMint!: PublicKey;
   redeemableMintDecimals!: number; // u16
+  isFrozen!: boolean;
   redeemableGlobalSupplyCap!: BN; // u128
   redeemableCirculatingSupply!: BN; // u128
+  registeredMercurialVaultDepositoriesCount!: number; // u8
+  registeredCredixLpDepositoriesCount!: number; // u8
```
Comment (crypto-vincent) [inline src/interfaces.ts:19]: same order as controller.rs

Comment (Orelsanpls) [reply src/interfaces.ts:19]: Is it this controller version? (the one in the main branch)

https://github.com/UXDProtocol/uxd-program/blob/a78cff4426625627e10a21ca9d7887c3cbcd6131/target/idl/uxd.json#L2016

If yes, do we want to add all missing informations? Like `registeredMercurialVaultDepositories`

Comment (crypto-vincent) [reply src/interfaces.ts:19]: yeah this is the controller account's fields in order they are declared in the IDL

Comment (crypto-vincent) [reply src/interfaces.ts:19]: Ok 👌  adding the registered depositories fields

Comment (crypto-vincent) [reply src/interfaces.ts:19]: Done!

**Review by Orelsanpls — APPROVED** [review]
Seems good :)

**Pull request merged**